packages/android_intent/pubspec.yaml

```diff
@@ -1,5 +1,5 @@
 name: android_intent
-description: A plugin for launching Intents on Android.
+description: Flutter plugin for launching Android Intents. Not supported on iOS.
```

Comment:
We mention iOS here, even though it is not supported. Is that what we want?

Reply:
I think it's OK to be explicit in this case